### PR TITLE
[deploy] Split Ambari plugin to independent repo

### LINKED ISSUE #7036: [Feature][deploy] Split Ambari plugin to independent repo
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

I submit mail thread: https://lists.apache.org/thread/5l2zy28q1l5tl87q84h8qsw59z5vg2ly about split Apache Ambari plugis to independent repo and it seem pass even though we have some misunderstand there. 

And we redeclare the core point here:

* Ambari plug-in is a deployment plugin and not update for two years,  and support DS 1.3.x. We DS 2.0 have quit different deployment way and it seem no support 2.0 anymore.
* Delete ambari plugin context for 2.0 and dev, cause it not work in 2.0
* We just split ambari plugin to independent repo(https://github.com/apache/dolphinscheduler-ambari), without delete and code and just remove it. Make sure we remove all things related to ambari. If someone who  interested in it, could submit a patch to support to 2.0(**And we hope someone could support it sincerely**).

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by zhongjiajie**
fix: #7036

### CODE REVIEW

File: dolphinscheduler-ui/pom.xml [hunk 1]
```diff
@@ -89,61 +89,5 @@
...
-    <profile>
-      <id>rpmbuild</id>
-      <build>
-        <plugins>
-          <plugin>
-            <groupId>com.github.eirslett</groupId>
-            <artifactId>frontend-maven-plugin</artifactId>
-            <version>${frontend-maven-plugin.version}</version>
-            <executions>
-              <execution>
```
Comment (CalvinKirs) [inline dolphinscheduler-ui/pom.xml:102]: Why delete this?

Comment (zhongjiajie) [reply dolphinscheduler-ui/pom.xml:102]: This profile it also work for ambari plugin, I can not found any others context link to this.

Comment (zhongjiajie) [reply dolphinscheduler-ui/pom.xml:102]: I compare two profile in ui/pom.xml, they are totally the same except profile name.

Comment (CalvinKirs) [reply dolphinscheduler-ui/pom.xml:102]: you are right~

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**